### DISCUSSION TITLE
Linux: parsing from a file local to grafana server

### DISCUSSION THREAD

**Discussion by bruteMax** [grafana/grafana-infinity-datasource#525] [Q&A]
Is this possible?  I don't know if there's a setting in grafana.ini, or some OS-based setting, that would enable Infinity to read in a file by specifying it as part of a URL source.

Something like this - http://localhost:3000/myFileToParse.xml

Any help is appreciated.  Thanks.

**Comment by yesoreyeram** [comment]
yea.. accessing via http://localhost is possible as long as grafana have access to that URL. ( but directly accessing file system is not allowed. Only the files exposed via any web server like nginx , apache can be used )
